### PR TITLE
Update python binding build docs

### PR DESCRIPTION
**Pull request by AllSeeingEyeTolledEweSew**
This should reflect how things currently work

### CODE REVIEW

File: docs/python_binding.rst [hunk 1]
```diff
@@ -11,86 +11,140 @@ libtorrent python binding
...
+	# the following will build with DEFAULT optimization
+	python setup.py install
 
-Note: If you are using 64 bits python you should edit setup.py and add this to the b2 command:
-``address-model=64``
+In order to customize the build *and* run other steps like installation, you
+should run the steps inline with ``build_ext``::
 
-This will create the file libtorrent.pyd inside build/lib/ that contains the binding.
+	python setup.py build_ext --optimization=space install
```
Comment (arvidn) [inline docs/python_binding.rst:103]: a few of these are outdated with the new `--b2-args=` argument

**Review by arvidn — CHANGES_REQUESTED** [review]
looks good. sorry for the delay. It should probably be updated to mention `--b2-args=` instead now though. I will back-port some of this to `RC_1_2`

**Comment by AllSeeingEyeTolledEweSew** [comment]
Should I just move these changes into #5948 and modify as appropriate?

**Comment by AllSeeingEyeTolledEweSew** [comment]
Done externally

**Pull request closed without merging**